elementary/monitor/alerts/alert.py

```diff
@@ -80,6 +85,15 @@ def consice_name(self):
         return "Alert"
 
 
+class ValidAlert(Alert):
```

Comment:
Care to clarify what's the purpose of this compared to `Alert`?

Reply:
Removed it. 
I added it only to simplify some of the Unions we did, but didn't actually need it.